akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/HttpCookie.scala

```diff
@@ -125,8 +134,9 @@ object HttpCookie {
     path:      Option[String]   = None,
     secure:    Boolean          = false,
     httpOnly:  Boolean          = false,
+    sameSite:  Option[SameSite] = None,
```

Comment:
If we move it to the last position, it will probably keep binary compat for the default value. We'll still need a deprecated version of the old one. Or we just deprecate the version with the default arguments, saying that people should use `fromPair(pair).withABC.withDEF`.